css/media.css

```diff
@@ -0,0 +1,256 @@
+@media (max-width:1280px) {
+
+    .oranges_boxes_general {
+        display: flex;   
+        background-image: url();
+    }
+    
+    .oranges_boxes {
+        width: 100%;
+        overflow: hidden;
+    }
+  
+    .farm img {
+     display: none;
+    }   
+    .farm_box {
+        width: 90%;
+        position: unset;
+    }
+    .farm {
+        position: unset;
+    }
+
+} 
+
+@media (max-width:1100px) {
+
+.nav li {
+    margin: 0 10px;
+}
+.subscribe_box_left h2 {
+    font-size: 40px;
+    line-height: 45px;
+ }
+
```

Comment:
.vegetables_box is a very big card look like so much - fix is add new style in media-request

Reply:
.choice_item is very big too - do less and set in one row